docs/mux-logging-service.md

```diff
@@ -41,30 +41,27 @@ There are environment variables for the logging-mux DC config:
   on each node.  This allows the per-node Fluentd collectors to read and ship
   logs off the node as fast as possible to the logging-mux service, which will
   normalize and enrich with Kubernetes metadata, and store in Elasticsearch.
-* `MUX_ALLOW_EXTERNAL` - `true`/`false` - default is `false` - if `true`, the
-  logging-mux service will be able to handle log records sent from outside of
-  the cluster, as described below.
+  This will also be able to process records sent from outside of the cluster,
+  if the service has exposed an externalIP.
 * `FORWARD_LISTEN_PORT` - default `24284` - port to listen for secure_forward
   protocol log messages
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
 These are environment variables for the logging-fluentd daemonset config:
 
-* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
-  will not send logs directly to Elasticsearch, instead it will use
-  `secure_forward` to send logs to mux.
-* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset - if
-  `minimal`, this is the same behavior as `USE_MUX_CLIENT true`.  If
-  `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
+* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset
+  If `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
   and filtering *except* the Kubernetes metadata annotation.  The records will
   be sent to mux for Kubernetes metadata annotation before sending the logs to
   Elasticsearch.  In this case, it is assumed you want to deploy mux to be as
   lightweight as possible, and move the processing burden to the individual
```

Comment:
Perhaps word the last part, "and move as much of the processing burden as possible to the individual fluentd collector pods."